htmlcov/calculator_py.html

```diff
@@ -0,0 +1,107 @@
+
```

Comment:
for this attempt i should have been clear to only show your mentor the html coverage report rather than upload it to github directly. the reason for this is that these files change a lot but are generated - we usually don't put generated code in the repo cause it confuses readers in pull requests.

do a git rm here